src/main/java/edu/ie3/datamodel/models/input/system/type/chargingpoint/ChargingPointTypeUtils.java

```diff
@@ -0,0 +1,246 @@
+          DC,
+          new HashSet<>(Arrays.asList("tesla1", "teslav1", "supercharger1", "supercharger")));
+
+  public static final ChargingPointType TeslaSuperChargerV2 =
+      new ChargingPointType(
+          "TeslaSuperChargerV2",
+          Quantities.getQuantity(150, StandardUnits.S_RATED),
+          DC,
+          new HashSet<>(Arrays.asList("tesla2", "teslav2", "supercharger2")));
+
+  public static final ChargingPointType TeslaSuperChargerV3 =
+      new ChargingPointType(
+          "TeslaSuperChargerV3",
+          Quantities.getQuantity(250, StandardUnits.S_RATED),
+          DC,
+          new HashSet<>(Arrays.asList("tesla3", "teslav3", "supercharger3")));
+
+  protected static final Map<String, ChargingPointType> commonChargingPointTypes =
+      Collections.unmodifiableMap(
+          Stream.of(
+                  HouseholdSocket,
+                  BlueHouseholdSocket,
+                  Cee16ASocket,
+                  Cee32ASocket,
+                  Cee63ASocket,
+                  ChargingStationType1,
+                  ChargingStationType2,
+                  ChargingStationCcsComboType1,
+                  ChargingStationCcsComboType2,
+                  TeslaSuperChargerV1,
+                  TeslaSuperChargerV2,
+                  TeslaSuperChargerV3)
+              .flatMap(
+                  type ->
+                      Stream.concat(
+                          Stream.of(type.getId().toLowerCase()),
+                          type.getSynonymousIds().stream().map(String::toLowerCase))
+                          .collect(Collectors.toMap(Function.identity(), v -> type)).entrySet()
+                          .stream())
+              .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue)));
+
+  /**
+   * Parse a given string into either a custom {@link ChargingPointType} or a standard {@link
+   * ChargingPointType}. A custom charging point type can be created if the prsable string matches
+   * the following regex {@code (\w+\d*)\s*\(\s*(\d+\.?\d+)\s*\|\s*(AC|DC)\s*\)} e.g. {@code
+   * FastCharger(50|DC)} is a valid custom charging point definition. The apparent power value is
+   * expected to be in kVA.
+   *
+   * <p>For all available common standard charging point types see {@link
+   * #commonChargingPointTypes}.
+   *
+   * @param parsableString the string to be parsed. Either a valid custom string or the id of a
+   *     common standard charging point
+   * @return instance of a {@link ChargingPointType}
+   * @throws ChargingPointTypeException if the string is neither a valid custom charging point
+   *     string nor a valid common standard charging point id
+   */
+  public static ChargingPointType parse(String parsableString) throws ChargingPointTypeException {
+
+    ChargingPointType res;
+
+    // valid regex for either custom or pre-defined types
+    String validCustomRegex = "(\\w+\\d*)\\s*\\(\\s*(\\d+\\.?\\d+)\\s*\\|\\s*(AC|DC)\\s*\\)";
```

Comment:
See comment / suggestion in documentation.

Reply:
see [answer](https://github.com/ie3-institute/PowerSystemDataModel/pull/196#discussion_r504565700)